### PR TITLE
changed \sec to \S

### PR DESCRIPTION
**Pull request by rohith1125**
Removed the duplicate, and changed /sec to /S

### CODE REVIEW

File: dictionary.txt [hunk 1]
```diff
@@ -880,5 +880,5 @@ _rp	₎
 \hslash	ℏ	
 \dagger	†	
 \ddagger	‡	
-\smalltilde	˜	
+\smalltilde	˜
```
Comment (DenverCoder1) [inline dictionary.txt:883]: Fix the missing tab

```suggestion
\smalltilde	˜	
```

**Comment by DenverCoder1** [comment]
Looks good. Thanks for contributing! :tada:

**Comment by rohith1125** [comment]
Thank you so much! Sorry for messing up 😓 I'm kinda new to raising PR's

**Comment by DenverCoder1** [comment]
Not a problem. The more you contribute, the more you can improve! :+1:

**Pull request merged**